core/src/cubos/core/io/glfw_window.cpp

```diff
@@ -175,6 +175,9 @@ void GLFWWindow::setMouseState(MouseState state)
     case MouseState::Hidden:
         cursorState = GLFW_CURSOR_HIDDEN;
         break;
+    default:
+        CUBOS_ERROR("Unknown MouseState, doing nothing");
```

Comment:
Since it doesn't halt further code execution, it should be a warning instead
```suggestion
        CUBOS_WARNING("Unknown MouseState, doing nothing");
```